app/views/taxon_names/batch_load/castor/_batch_load.html.erb

```diff
@@ -0,0 +1,39 @@
+<div>
+  <h2> Castor batch load </h2>
```

Comment:
Very cool.  Might be a candidate for renaming to a more generalized name, but maybe we just know it as the "Castor" version.